### PR TITLE
remove left over code from monero

### PR DESCRIPTION
**Pull request by psychocrypt**
Remove unused variables those was used in code which handles tx version one.

### CODE REVIEW

File: src/cryptonote_core/blockchain.cpp [hunk 1]
```diff
@@ -407,10 +407,6 @@ bool Blockchain::init(BlockchainDB *db, const network_type nettype, bool offline
 	uint64_t top_block_timestamp = m_db->get_top_block_timestamp();
 	uint64_t timestamp_diff = time(NULL) - top_block_timestamp;
 
-	// genesis block has no timestamp, could probably change it to have timestamp of 1341378000...
```
Comment (fireice-uk) [inline src/cryptonote_core/blockchain.cpp]: Why is this part removed?

Comment (psychocrypt) [reply src/cryptonote_core/blockchain.cpp]: I assumend this was a left over from monero and the genesis sumokoin block has an time stamp. I am wrong?

Comment (fireice-uk) [reply src/cryptonote_core/blockchain.cpp]: No, it also has the timestamp set to 0

Comment (psychocrypt) [reply src/cryptonote_core/blockchain.cpp]: OK this change is wrong, The genesis block has time stamp is zero. I will remove this change.
